source/extensions/filters/http/istio_stats/config.proto

```diff
@@ -111,4 +111,10 @@ message PluginConfig {
 
   // Proxy deployment type.
   Reporter reporter = 10;
+
+  // Metric expiry duration. Defaults to 12h. Set to 0 to disable the metric expiry.
+  google.protobuf.Duration expiry_duration = 11;
+
+  // Metric expiry graceful drain duration. Defaults to 5m. No-op if the metric expiry is disabled.
+  google.protobuf.Duration graceful_drain_duration = 12;
```

Comment:
- What does draining do?
- As graceful_drain_duration is used as drain_interval_ms,  should graceful_drain_duration be called graceful_drain_interval?

Reply:
This is not HTTP drain, this is metrics drain. See the latest code comments.